app/client/sagas/creditTransferSagas.js

```diff
@@ -2,18 +2,27 @@ import { put, takeEvery, call, all } from "redux-saga/effects";
 import { normalize } from "normalizr";
 
 import {
-  PUSHER_CREDIT_TRANSFER,
-  UPDATE_CREDIT_TRANSFER_LIST,
-  LOAD_CREDIT_TRANSFER_LIST_REQUEST,
-  LOAD_CREDIT_TRANSFER_LIST_SUCCESS,
-  LOAD_CREDIT_TRANSFER_LIST_FAILURE,
-  MODIFY_TRANSFER_FAILURE,
-  MODIFY_TRANSFER_REQUEST,
-  MODIFY_TRANSFER_SUCCESS,
-  CREATE_TRANSFER_REQUEST,
-  CREATE_TRANSFER_SUCCESS,
-  CREATE_TRANSFER_FAILURE
-} from "../reducers/creditTransferReducer.js";
+  PUSHER_CREDIT_TRANSFER // TODO UNUSED?
```

Comment:
`PUSHER_CREDIT_TRANSFER` is used to handle credit transfers received from the Pusher WebSocket on the dashboard page.

https://github.com/teamsempo/SempoBlockchain/blob/7588356216d645fbf3a998821d6f70215b0cccb7/app/client/components/pages/dashboardPage.jsx#L6

https://github.com/teamsempo/SempoBlockchain/blob/7588356216d645fbf3a998821d6f70215b0cccb7/app/client/components/pages/dashboardPage.jsx#L91-L116

Reply:
`PUSHER_CREDIT_TRANSFER` cannot be resolved here so is causing the following error when testing frontend:

```
Error: takeEvery$1 requires a pattern or channel
    at check (io-6de156f3.js?86aa:33)
    at validateTakeEffect (redux-saga-effects.esm.js?fc1c:382)
    at takeEvery$1 (redux-saga-effects.esm.js?fc1c:388)
    at watchPusherCreditTransfer$ (creditTransferSagas.js?baa4:135)
    at tryCatch (runtime.js?f3a2:65)
    at Generator.invoke [as _invoke] (runtime.js?f3a2:303)
    at Generator.prototype.<computed> [as next] (runtime.js?f3a2:117)
    at next (redux-saga-core.esm.js?42a9:1157)
    at proc (redux-saga-core.esm.js?42a9:1108)
    at runEffect (redux-saga-core.esm.js?42a9:1199)
```

Reply:
Oops - fixed now.  Not sure how I missed that.

So how are events being pushed into this websocket subscription?

Reply:
They're pushed to the frontend after a successful request:
https://github.com/teamsempo/SempoBlockchain/blob/7588356216d645fbf3a998821d6f70215b0cccb7/app/server/__init__.py#L115-L132